src/main/g8/project/Versions.scala

```diff
@@ -1,8 +1,8 @@
 object Versions {
-  val akkaV = "2.5.4"
-  val akkaHttpV = "10.0.10"
+  val akkaV = "2.5.11"
+  val akkaHttpV = "10.1.0"
   val squbsV = "0.10.0-SNAPSHOT"
   val logbackClassicV = "1.1.3"
-  val json4sV = "3.5.0"
-  val akkaHttpJson4sV = "1.11.0"
+  val json4sV = "3.5.3"
+  val akkaHttpJson4sV = "1.20.0"
```

Comment:
This seems to be 1 version ahead of squbs open source. But I think this should be just fine.